File: tests/interfile/csharp/sanitiser_active/Senders.cs
class Senders
{
    public static void SendClean(string value)
    {
        string safe = sanitize(value);
        // ok: sanitiser-active-csharp
        sink(safe);
    }

    public static void SendDirty(string value)
    {
        // ruleid: sanitiser-active-csharp
        sink(value);
    }
}

File: tests/interfile/java/sanitiser_active/Senders.java
class Senders {
    static void sendClean(String value) {
        String safe = sanitize(value);
        // ok: sanitiser-active-java
        sink(safe);
    }

    static void sendDirty(String value) {
        // ruleid: sanitiser-active-java
        sink(value);
    }
}
